### PR TITLE
adding Kahve BOT presence

### PR DESCRIPTION
**Pull request by eren885**
3lwO
utpH
RmAs
O9iW
2Mea
NXUF
Lkgb
XIYn

### CODE REVIEW

File: Kahve BOT/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "author": {
+        "name": "Eren",
+        "id": "478937019758673931"
+    },
+    "service": "Kahve",
```
Comment (Bas950) [inline Kahve BOT/dist/metadata.json]: Service name must be the same as the folder name.

Comment (Timeraa) [reply Kahve BOT/dist/metadata.json]: Eh no

Comment (Bas950) [reply Kahve BOT/dist/metadata.json]: Was that not needed anymore?

Comment (Bas950) [reply Kahve BOT/dist/metadata.json]: I thought it was still needed due to the source code button on the presence page.

Comment (Timeraa) [reply Kahve BOT/dist/metadata.json]: I mean try out and tell @Voknehzyr if its still broken

**Pull request merged**